posthog/models/async_deletion/delete_events.py

```diff
@@ -69,11 +70,22 @@ def process(self, deletions: list[AsyncDeletion]):
             # If the query size is greater than the max predicate size, execute the query and reset the query predicate
             if query_size > MAX_QUERY_SIZE:
                 logger.debug(f"Executing query with args: {args}")
-                sync_execute(
-                    query,
-                    args,
-                    settings={},
-                )
+                try:
+                    sync_execute(
+                        query,
+                        args,
+                        settings={},
+                    )
+                except Exception as e:
+                    # This is unfortunately needed because currently all lightweight deletes are executed sync
+                    if isinstance(e, SocketTimeoutError):
```

Comment:
Out of curiosity, couldn't you just `except SocketTimeError`?

Reply:
lol you could, I'm just clumsy with python

Reply:
updated to pattern match on `SocketTimeError`!